Formula/duff.rb

```diff
@@ -17,4 +17,17 @@ def install
                           "--mandir=#{man}"
     system "make", "install"
   end
+
+  test do
+    expected = <<-EOS.undent
+      2 files in cluster 1 (6 bytes, digest 8843d7f92416211de9ebb963ff4ce28125932878)
+      cmp1
+      cmp2
+    EOS
+
+    (testpath/"cmp1").write "foobar"
+    (testpath/"cmp2").write "foobar"
+
+    assert_equal expected, `"#{bin}/duff" cmp1 cmp2`
```

Comment:
We shouldn't be using backtics in tests directly. You want to use `shell_output` or `pipe_output` depending on circumstance.